library/core/src/task/wake.rs

```diff
@@ -63,15 +63,15 @@ impl RawWaker {
     /// Gets the `data` pointer used to create this `RawWaker`.
     #[inline]
     #[must_use]
-    #[unstable(feature = "waker_getters", issue = "96992")]
+    #[stable(feature = "waker_getters", since = "CURRENT_RUSTC_VERSION")]
     pub fn data(&self) -> *const () {
         self.data
     }
 
     /// Gets the `vtable` pointer used to create this `RawWaker`.
     #[inline]
     #[must_use]
-    #[unstable(feature = "waker_getters", issue = "96992")]
+    #[stable(feature = "waker_getters", since = "CURRENT_RUSTC_VERSION")]
     pub fn vtable(&self) -> &'static RawWakerVTable {
         self.vtable
     }
```

Comment:
The API still needs to be updated to what was decided via FCP here:

- https://github.com/rust-lang/rust/issues/96992#issuecomment-1941998046

Reply:
Updated.

Should the new `pub const unsafe fn new(data: *const (), vtable: &'static RawWakerVTable) -> Self;`  constructor from that comment be made immediately stable in this PR? Or should that be done separately under a new feature since it's not really part of `waker_getters`? The plan to deprecate `RawWaker` doesn't work because it's in the `RawWakerVTable` signature as discussed in later comments on the tracking issue.

Reply:
cc @dtolnay

Reply:
`Waker::new` is an obvious function with an obvious name, and is the composition of 2 already-stable APIs (`RawWaker::new` + `Waker::from_raw`). It was discussed by most of the team in a library API team meeting (https://github.com/rust-lang/rust/issues/96992#issuecomment-1941973331) and approved by the team's consensus process in https://github.com/rust-lang/rust/issues/96992#issuecomment-1941998046 with no objections. I think the circumstances are fine to stabilize this immediately.